### PR TITLE
Docs #982: Fix tiered storage commands

### LINKED ISSUE #982: Separate tiered storage commands
### Describe the Issue

Due to the current behaviour outlined in https://github.com/redpanda-data/redpanda/issues/4499 , these commands need to be issued separately, rather than in a one-liner.

### Updates to existing documentation

https://docs.redpanda.com/docs/platform/data-management/tiered-storage/#enable-tiered-storage-for-a-topic

Instead of specifying:

```
rpk topic alter-config <topic_name> --set redpanda.remote.read=true --set redpanda.remote.write=true
```

It should currently be two commands of:

```
rpk topic alter-config <topic_name> --set redpanda.remote.read=true
```

and 

```
rpk topic alter-config <topic_name> --set redpanda.remote.write=true
```

### Additional notes

When the linked core issue is resolved, it would be neater to revert this change, however having 2 commands in that case would still be valid, so it's not vital to revert.

### PR DESCRIPTION
**Pull request by JFlath**
Fixes #982

### CODE REVIEW

**Review by shorgi — APPROVED** [review]

**Comment by Feediver1** [comment]
Just an FYI for @micheleRP as it touches Tiered Storage.

**Comment by Deflaimun** [comment]
Hi @JFlath. I noticed that the latest page was not updated. I went ahead and created the update. Could you review to see if it's correct? Thanks!

Also @shorgi please run another review round.

**Comment by Feediver1** [comment]
@JFlath @shorgi Can you help me understand what stalled the merge of this PR? Is more info needed? Further review?

**Comment by Feediver1** [comment]
@micheleRP Already included elsewhere in one of your recent PRs?

**Review by shorgi — APPROVED** [review]
LGTM, my apologies for not re-approving earlier

File: versioned_docs/version-21.11/data-management/tiered-storage.mdx [hunk 1]
```diff
@@ -118,10 +118,14 @@ To enable Tiered Storage on a topic, you can set the `redpanda.remote.write` and
 rpk topic create <topic_name> -c redpanda.remote.read=true -c redpanda.remote.write=true
 ```
 
-And use this command to enable Tiered Storage on an existing topic: 
+And use both these commands to enable Tiered Storage on an existing topic:
 
+```bash
+rpk topic alter-config <topic_name> --set redpanda.remote.read=true
 ```
```
[the remark this answers is not in the document]
Comment (Deflaimun) [reply versioned_docs/version-21.11/data-management/tiered-storage.mdx:125]: To tie two commands in linux you must add `&&` so this should be 

```suggestion
```bash
rpk topic alter-config <topic_name> --set redpanda.remote.read=true &&
rpk topic alter-config <topic_name> --set redpanda.remote.write=true
```
```

**Pull request merged**